Automatic Validation ended with:
> 2024-02-02T21:40:49.8920469Z ##[error] One or more errors occurred. (SQL error or missing database (0x87AF0001)) SQL error or missing database (0x87AF0001)

(Automated response - build 695.)

--- a/manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.installer.yaml
+++ b/manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.installer.yaml
@@ -8,9 +8,13 @@ Installers:
   Architecture: x64
   InstallerType: inno
   InstallerSha256: 03C7A64C9616D38BB21891B7F9C976734C2994C9A127AFD75FFF396F6037476A
+  AppsAndFeaturesEntries:
+  - DisplayName: Stella 6.7.1
 - InstallerUrl: https://github.com/stella-emu/stella/releases/download/6.7.1/Stella-6.7.1-win32.exe
   Architecture: x86
   InstallerType: inno
+  AppsAndFeaturesEntries:
+  - DisplayName: Stella 6.7.1
   InstallerSha256: DB2DACAC3F3C047987989C20B1A630FB18B97A431D120A1591187F0DD1CEEAF9
 - InstallerUrl: https://github.com/stella-emu/stella/releases/download/6.7.1/Stella-6.7.1-windows.zip
   Architecture: x86

--- a/manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.installer.yaml
+++ b/manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.installer.yaml
@@ -10,12 +10,18 @@ Installers:
   InstallerSha256: 03C7A64C9616D38BB21891B7F9C976734C2994C9A127AFD75FFF396F6037476A
   AppsAndFeaturesEntries:
   - DisplayName: Stella 6.7.1
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
 - InstallerUrl: https://github.com/stella-emu/stella/releases/download/6.7.1/Stella-6.7.1-win32.exe
   Architecture: x86
   InstallerType: inno
   AppsAndFeaturesEntries:
   - DisplayName: Stella 6.7.1
   InstallerSha256: DB2DACAC3F3C047987989C20B1A630FB18B97A431D120A1591187F0DD1CEEAF9
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
 - InstallerUrl: https://github.com/stella-emu/stella/releases/download/6.7.1/Stella-6.7.1-windows.zip
   Architecture: x86
   InstallerType: zip
@@ -23,6 +29,9 @@ Installers:
   NestedInstallerFiles:
   - RelativeFilePath: Stella-6.7.1\32-bit\Stella.exe
   InstallerSha256: 6781EFFF51AA28390B2F858A1E7B8B98111FAD62BCCEDE400C6E741B43A49099
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
 - InstallerUrl: https://github.com/stella-emu/stella/releases/download/6.7.1/Stella-6.7.1-windows.zip
   Architecture: x64
   InstallerType: zip
@@ -30,5 +39,8 @@ Installers:
   NestedInstallerFiles:
   - RelativeFilePath: Stella-6.7.1\64-bit\Stella.exe
   InstallerSha256: 6781EFFF51AA28390B2F858A1E7B8B98111FAD62BCCEDE400C6E741B43A49099
+  Dependencies:
+    PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
 ManifestType: installer
 ManifestVersion: 1.5.0

--- a/manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.locale.en-US.yaml
+++ b/manifests/t/TheStellaTeam/Stella/6.7.1/TheStellaTeam.Stella.locale.en-US.yaml
@@ -8,6 +8,6 @@ Publisher: The Stella Team
 PackageName: Stella
 License: GPL-2.0
 Copyright: Copyright © 1995-2024 Bradford W. Mott, Stephen Anthony and The Stella Team
-ShortDescription: Stella Setup
+ShortDescription: A multi-platform Atari 2600 Emulator.
 ManifestType: defaultLocale
 ManifestVersion: 1.5.0

